freebridge-ai/routers/recommendation.py

```
@@ -28,36 +31,114 @@ def get_vs():
+
+        all_context = "\n\n".join([d.page_content for d in (experienced_docs[:5] + newbie_docs[:2])])
+
+        prompt = ChatPromptTemplate.from_template("""
+        전문 헤드헌터로서 다음 유저 중 공고에 가장 적합한 7명을 추천하세요.
+        (숙련자 5명, 신입/신규 2명을 가급적 포함하되 적합도가 중요함)
+        <context>{context}</context>
+        """)
+        
+        result = await structured_llm.ainvoke(prompt.format(context=all_context))
+        return {"success": True, "data": result.matches[:7]}
+    except Exception as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)